### ISSUE TITLE
Datepicker keyboard navigation

### ISSUE THREAD

**Issue by laiseing** [laboratoriobridge/bold#461]
When the calendar is visible, a user should be able to access any of the controls (e.g. dates, month/year selection, previous/next month/year, Today button, etc.) with keyboard navigation.
